### DISCUSSION TITLE
Yet Another Multiple Datasources with Prisma Question

### DISCUSSION THREAD

**Discussion by mikecfisher** [prisma/prisma#3605] [Q&A] [answered]
Hi all, I'm working on a project where I need to communicate with a postgres database. I really want to use Prisma. However it seems Prisma doesn't support communicating with 2 databases (in my case 2 Postgres DBs) I absolutely need to talk to both of them to make my feature work though.

Is there any workaround for this in the time being or can I simply just not use Prisma right now??

**Comment by ryands17** [comment, marked as answer]
Hey @mikefisher84 👋 
You can use [this](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) workaround as decribed in this issue for the timebeing.

Let me know if this works for you.

**Reply by mikecfisher** [reply]
Thank you @ryands17 ! While I agree that solution isn't ideal its totally doable. I searched high and low for this solution and wasn't able to find it. I'd suggest adding this to the docs somewhere because I was inches away from not using Prisma for this project until I found this. 

Thanks again!

**Reply by ryands17** [reply]
@mikefisher84 👋 
Do let me know if you face any issues implementing this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
